reference/5.1/PowershellGet/Find-Module.md

```diff
@@ -16,67 +16,84 @@ Finds modules from an online gallery that match specified criteria.
 
 ## SYNTAX
 
+### __AllParameterSets
```

Comment:
Change the parameter set name to "All".

Reply:
Thanks for the feedback. I copyedited the Find-Module document and will push the changes to update the PR.